src/Migrator.EF6.Tools/CommandBase.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NET451
+﻿#if NET462
```

Comment:
I think retargeting to NET46 is favorable yes, to support the minimum version possible.